src/main/resources/db/migration/V8_alter_notes_sequence.sql

```diff
@@ -0,0 +1 @@
+ALTER SEQUENCE notes_seq INCREMENT BY 1;
```

Comment:
Try not to divide patches too much. You'll end up with the unmaintainable amount of them.
You can group your current ones into 2 maybe 3 patches